### PR TITLE
feat: add Create Layer and Add Layer To Map activities

### PR DESCRIPTION
**Pull request by rcooney**

### CODE REVIEW

File: src/activities/CreateLayerFromArcGisServerUrl.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     * @displayName URL
+     * @description The URL to the ArcGIS Server service.
+     * @required
+     */
+    url: string;
+
+    /**
+     * @description Any of the layer's properties for constructing the layer instance (e.g. popupTemplate, renderer, etc.).
+     */
+    properties?: any;
```
Comment (cdoak-gh) [inline src/activities/CreateLayerFromArcGisServerUrl.ts:15]: Do layer properties have a type?

**Review by cdoak-gh — COMMENTED** [review]
Wondering if it's possible to type the properties input on CreateLayerFromArgisServerUrl

File: src/activities/CreateLayerFromArcGisServerUrl.ts (hunk 1 above)
Comment (rcooney) [reply src/activities/CreateLayerFromArcGisServerUrl.ts:15]: Not in the type definitions for the method we are using
image

That said, that method does just call the correct layer constructor, so there are a handful of interfaces, like `FeatureLayerProperties`, `ArcGISCachedServiceProperties`, `ArcGISImageServiceProperties`, `ArcGISMapServiceProperties`, etc., that must be used in practice. The trouble is that there are so many of them and they each have loads of properties. We would have to supply the superset of all possibilities. At which point, it is basically `any`.

Comment (cdoak-gh) [reply src/activities/CreateLayerFromArcGisServerUrl.ts:15]: I figured it must be something like that

**Review by cdoak-gh — APPROVED** [review]

**Pull request merged**